sql_generators/glean_usage/event_monitoring_live.py

```diff
@@ -35,6 +39,21 @@ def __init__(self) -> None:
         self.custom_render_kwargs = {}
         self.base_table_name = "events_v1"
 
+    def _get_tables_with_events(self, v1_name: str) -> Set[str]:
+        pings = set()
+        resp = requests.get(METRICS_INFO_URL.format(app_name=v1_name))
+        if resp.status_code == 404:
+            return pings
+        resp.raise_for_status()
+        metrics_json = resp.json()
+
+        for _, metric in metrics_json.items():
+            if metric.get("type", None) == "event":
+                latest_history = metric.get("history", [])[-1]
+                pings.update(latest_history.get("send_in_pings", []))
```

Comment:
This might be an edge not worth considering but only looking at the pings in the latest revision could miss events from older versions of the app if a ping doesn't have any events in the latest version.  It might be safe to ignore those though, just want to make sure that's considered

Reply:
There were some cases where pings got seemingly removed. Older history entries were referring to pings that (no longer?) had a table in BigQuery. Using the latest history entry should account for deprecated pings.

Example: There are some events for the [firefox_desktop_background_tasks](https://probeinfo.telemetry.mozilla.org/glean/firefox-desktop-background-tasks/metrics) that got sent as part of `remove-directory` and later as part of `background-tasks` pings. There is no `remove_directory` table.

Reply:
Maybe it was never added to pings.yaml.  I'm not sure if that would cause errors somewhere.  Ok, makes sense to use the latest